websites/docs-smoke-test/src/content/views/code-blocks.mdx

```diff
@@ -1,5 +1,7 @@
 ---
 title: Code Blocks
+showTimeToRead: true
```

Comment:
The test cases you put into the pages are all just covering the same "good cases".   You should have pages that test unlogical scenarios, too.   E.g. setting a manual time but not showing it or setting something invalid as the frontmatter.